src/assets/styles/style.less

```diff
@@ -39,23 +46,20 @@ body {
 
 #showcase-title {
   color: @chart-teal;
-  font-size: 20px;
   position: absolute;
   right: 75px;
   z-index: 10;
-  top: -12px;
+  top: -10px;
```

Comment:
Yes; I removed the `font-size` here because it wasn't actually applying to any fonts (they're specified in the children), but was adding extra padding to the text that I needed to take into account.